docs/source/quickstart.rst

```diff
@@ -1,88 +1,110 @@
 .. _quickstart:
 
-Quickstart
-==========
+Quickstart: Install MLflow, instrument code & view results in minutes
+======================================================================
 
-Installing MLflow
+
+In 15-20 minutes, you will:
+
+- Install MLflow
+- Add MLflow tracking to your code
+- View run metrics in the MLflow tracking UI
+- (Optional) Run a tracking server to share results with others
+- (Optional) Use Databricks to store your results
+- Store the models produced by your runs
+- Run a model from a previous run
```

Comment:
Nit: We typically refer to "model loading" and "model inference", rather than "running a model"

```suggestion
- Load a model from a previous run for inference
```